.github/workflows/test-build-binaries.yml

```
@@ -458,6 +498,29 @@ jobs:
+    if: startsWith(github.ref, 'refs/tags/')
+    steps:
+      - name: Get LedFx Version
+        id: ledfx-version
+        run: |
+          echo "ledfx-version=$(docker run --rm --entrypoint ledfx ghcr.io/${{ github.repository_owner }}/ledfx:edge --version | sed 's/LedFx //')" >> $GITHUB_OUTPUT
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)